kimchi/src/oracles.rs

```diff
@@ -20,8 +20,6 @@ where
     pub oracles: RandomOracles<G::ScalarField>,
     /// the computed powers of alpha
     pub all_alphas: Alphas<G::ScalarField>,
-    /// public polynomial evaluations
-    pub public_evals: [Vec<G::ScalarField>; 2],
```

Comment:
So, since the public polynomials are now inside ProofEvaluations they no longer contribute to Fiat Shamir? Meaning, now the verifier does not compute the public polynomial evaluations? Why is this? Efficiency?

Reply:
They're already in the evaluations; we don't need to compute them again.